hydraw/src/Hydra/Painter.hs

```diff
@@ -39,6 +39,7 @@ paintPixel signingKeyPath cnx pixel = do
       case mkPaintTx (txIn, txOut) (myAddress, txOutValue txOut) sk pixel of
         Left err -> error $ "failed to build pixel transaction " <> show err
         Right tx -> sendTextData cnx $ Aeson.encode $ NewTx tx
+    Right other -> error $ "Unexpected server answer:  " <> decodeUtf8 msg
```

Comment:
do we have only warnings for these unmatch cases?
i think we should prevent the CI from building in such situations.

Reply:
This package is not built & tested in the CI workflow right now.